tools/scripts/bootstrap.py

```diff
@@ -41,369 +41,395 @@
-            soname, _ = self.__read_elf(lib_path)
-            adjusted_soname = self.__adjust_soname(soname)
-            if (adjusted_soname != soname):
-                self.__soname_replacements.append((soname, adjusted_soname))
-            if not install_list or lib_path in install_list:
-                shutil.copy(lib_path, os.path.join(lib_dir, adjusted_soname))
+            soname, _ = self._read_elf(lib_path)
+            adjusted_soname = self._adjust_soname(soname)
+            if adjusted_soname != soname:
+                self._soname_replacements.append((soname, adjusted_soname))
+            shutil.copyfile(lib_path, os.path.join(target_lib_dir, adjusted_soname))
 
-        for pair in self.__soname_replacements:
-            assert len(pair[0]) == len(pair[1])
+        for pair in self._soname_replacements:
+            assert(len(pair[0]) == len(pair[1]))
 
-        for lib_path in Path(lib_dir).glob('*.so'):
-            self.__replace_soname_values(lib_path)
+        for lib_path in Path(target_lib_dir).glob('*.so'):
+            self._replace_soname_values(lib_path)
+
+    def _copy_jni_libs(self, src_lib_dir, target_jni_lib_dir):
+        shutil.rmtree(target_jni_lib_dir, True)
+        os.makedirs(target_jni_lib_dir)
 
-    def __copy_jni_libs(self, jni_lib_dir, lib_dir, libs_paths = None):
-        if libs_paths is None:
-            if os.path.exists(jni_lib_dir):
-                shutil.rmtree(jni_lib_dir)
-            os.makedirs(jni_lib_dir)
-            libs_paths = list(Path(lib_dir).glob('*.so'))
-            libs_paths.extend(list(Path(os.path.join(lib_dir, 'wpe-webkit-1.0')).glob('*.so')))
-            libs_paths.extend(list(Path(os.path.join(lib_dir, 'wpe-webkit-1.0', 'injected-bundle')).glob('*.so')))
+        libs_paths = list(Path(src_lib_dir).glob('*.so'))
+        libs_paths.extend(list(Path(os.path.join(src_lib_dir, 'wpe-webkit-1.0')).glob('*.so')))
+        libs_paths.extend(list(Path(os.path.join(src_lib_dir, 'wpe-webkit-1.0', 'injected-bundle')).glob('*.so')))
 
         for lib_path in libs_paths:
-            if os.path.basename(lib_path) in self.__build_libs:
-                continue
-            shutil.copy(lib_path, os.path.join(jni_lib_dir, os.path.basename(lib_path)))
+            if os.path.basename(lib_path) not in self._build_libs:
+                shutil.copyfile(lib_path, os.path.join(target_jni_lib_dir, os.path.basename(lib_path)))
 
-    def __resolve_deps(self, lib_dir):
+    def _resolve_deps(self, lib_dir):
         soname_set = set()
-        needed_set = self.__base_needed
+        needed_set = set(self._base_needed)
 
         for lib_path in Path(lib_dir).glob('*.so'):
-            soname, needed_list = self.__read_elf(lib_path)
+            soname, needed_list = self._read_elf(lib_path)
             soname_set.update([soname])
             needed_set.update(needed_list)
 
-        print("NEEDED but not provided:")
+        print('NEEDED but not provided:')
```

Comment:
Also there's quite a bit of churn switching to from double-quote strings to single-quote strings, and PEP-8 does not advocate for neither... I would leave them as they were.

Reply:
I saw some strings with ' and some strings with " (without ' inside the string to justify the enclosing ").
So I decided to unify all strings. I used ' but I may have used " too, to be honest in Python it changes nothing.

As it may look strange to you, I will change them to " for all strings. The only important point is to be coherent and use the same for all strings.